resources/views/admin/orders/list.blade.php

```diff
@@ -26,13 +26,9 @@
 				fill: false,
 				
 				data: [
-					{{ $weekly_invoices[1] }},
-					{{ $weekly_invoices[2] }},
-					{{ $weekly_invoices[3] }},
-					{{ $weekly_invoices[4] }},
-					{{ $weekly_invoices[5] }},
-					{{ $weekly_invoices[6] }},
-					{{ $weekly_invoices[7] }}
+					@foreach($weekly_invoices as $invoice)
```

Comment:
Since we know that the array has elements 0 --> 6, we can just loop through it and inject the data.

Reply:
how are you using foreach without a @endforeach closing tag?